docs/cosmology/dev.rst

```diff
@@ -6,24 +6,22 @@ Cosmology For Developers
 Cosmologies in Functions
 ========================
 
-It is often useful to assume a default cosmology so that the exact
-cosmology does not have to be specified every time a function or method
-is called. In this case, it is possible to specify a "default"
-cosmology.
+It is often useful to assume a default cosmology so that the exact cosmology
+does not have to be specified every time a function or method is called. In
```

Comment:
I'd delete the last sentence (In this case, ...) - superfluous given what follows.

Reply:
Agreed.